src/wrapped/wrappedlibc.c

```diff
@@ -3811,6 +3811,13 @@ EXPORT int my_xdr_string(x64emu_t* emu, void* arg1, char** arg2, unsigned int ar
 #endif
 }
 
+EXPORT void my_xdr_free(x64emu_t* emu, void* arg1, char* arg2)
+{
+#ifdef HAVE_RPC_XDR_H
```

Comment:
what is this macro? were does it comes from?
my don't you use `my->xdr_free(arg1, arg2` instread, which will comes from tyhe lib openned.

Reply:
what happens if you don't have xdr stuffs and the programs use it?